exampleSite/config.toml

```diff
@@ -43,6 +43,8 @@ disqusShortname = "yourdiscussshortname"
     custom_css = []
 
 [taxonomies]
+  categories = "categories"
+  tags = "tags"
```

Comment:
Ok, reading the docs I found out that we need to redeclare the default taxonomies (tags and categories) when adding a custom one (series). So, you just need to change this file by adding tags and categories in the `singular = "plural"` form.

```
[taxonomies]
  category = "categories"
  series = "series"
  tag = "tags"
```